data/data-pipeline/data_pipeline/score/score_a.py

```diff
@@ -0,0 +1,13 @@
+from data_pipeline.score.score import Score, FN, pd, logger
```

Comment:
It seems a little odd to me we're not just directly importing pandas and the logger here – is there a reason we prefer to inherit them?

Reply:
That might be my lack of experience with python then. If it's more conventional to import them in every class that needs them than to inherit them, I'm fine with that.

Reply:
changed!